### PR TITLE
[id] Fixed Missed Translations

### PR DESCRIPTION
**Pull request by itsfaqih**
Apparently there're some missed translations and some words that I haven't exclude. So here's the fix, sorry for the inconvenience

### CODE REVIEW

File: excludes/id.php [hunk 1]
```diff
@@ -107,17 +116,19 @@
     'Rwanda',
     'Saint Barthelemy',
     'Saint Helena',
-    'Saint Kitts And Nevis',
+    'Saint Kitts and Nevis',
```
Comment (andrey-helldar) [inline excludes/id.php:119]: The keys to exclude are specified here.

`'Saint Kitts And Nevis' !== 'Saint Kitts and Nevis'`

See: https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L441

File: excludes/id.php [hunk 2]
```diff
@@ -107,17 +116,19 @@
...
     'Saint Lucia',
     'Saint Martin',
-    'Saint Pierre And Miquelon',
-    'Saint Vincent And Grenadines',
+    'Saint Pierre and Miquelon',
```
Comment (andrey-helldar) [inline excludes/id.php:122]: The keys to exclude are specified here.

`'Saint Pierre And Miquelon' !== 'Saint Pierre and Miquelon'`

See: https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L444

File: excludes/id.php [hunk 3]
```diff
@@ -107,17 +116,19 @@
...
+    'Saint Vincent and Grenadines',
```
Comment (andrey-helldar) [inline excludes/id.php:123]: The keys to exclude are specified here.

`'Saint Vincent And Grenadines' !== 'Saint Vincent and Grenadines'`

See: https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L445

File: excludes/id.php [hunk 4]
```diff
@@ -107,17 +116,19 @@
...
     'Samoa',
-    'San Marino',
+    'San MarinoID',
```
Comment (andrey-helldar) [inline excludes/id.php:125]: Invalid key name

File: excludes/id.php [hunk 5]
```diff
@@ -107,17 +116,19 @@
...
     'Senegal',
     'September',
+    'Serbia',
     'Seychelles',
     'Sierra Leone',
+    'Sint Maarten(Dutch part)',
```
Comment (andrey-helldar) [inline excludes/id.php:131]: Invalid key name.

Correct is `Sint Maarten (Dutch part)`

See: https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L470

File: excludes/id.php [hunk 6]
```diff
@@ -127,7 +138,7 @@
     'Tajikistan',
     'Tanzania',
     'Thailand',
-    'Timor-Leste',
+    'Timor - Leste',
```
Comment (andrey-helldar) [inline excludes/id.php:141]: Invalid key name.

Correct is `Timor-Leste`

See: https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L545

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Comment by andrey-helldar** [comment]
The exception file specifies keys, not values.

**Comment by itsfaqih** [comment]
Thank you for the review.
Just thought an idea, I think it'd be good if there's a test for checking whether a key is exists or not

**Comment by andrey-helldar** [comment]
> Thank you for the review.
> Just thought an idea, I think it'd be good if there's a test for checking whether a key is exists or not

We only test the localizations themselves.
Exception files work differently for us - each push to the `master` branch runs a script that checks the keys specified in the files from the `excludes` folder and automatically deletes nonexistent ones.

**Review by andrey-helldar — APPROVED** [review]

**Comment by caouecs** [comment]
thank you

**Pull request merged**